packages/ui/src/components/internal/Root.svelte

```diff
@@ -159,17 +163,12 @@
           <div class="clock">
             <Clock />
           </div>
-          <div class="flex-center widget">
-            <LangSelector />
-          </div>
-          <div class="flex-center widget cursor-pointer">
-            <ThemeSelector />
-          </div>
-          <div class="flex-center widget cursor-pointer">
+          <div class="flex-row-center gap-statusbar">
             <FontSizeSelector />
+            <ThemeSelector />
+            <LangSelector />
           </div>
-          <!-- svelte-ignore a11y-click-events-have-key-events -->
-          <div
```

Comment:
WiFi button used to show server statistics for us, we need another place in this case.